PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx

```diff
@@ -67,21 +67,21 @@ using std::endl;
  */
 AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA::AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA() :
 AliAnalysisTaskEmcalJet(),
-fUseRecalcPatches(false),
+fUseRecalcPatches(0x0),
```

Comment:
This is a change with no effect: The compiler will perform an implicit type conversion and replace the hex value with true. I recommend not to use numeric values like hex values for booleans as this gets confusing.